workspace.jsonc

```diff
@@ -38,16 +38,52 @@
      */
     "packageManager": "teambit.dependencies/pnpm",
     "policy": {
-      "dependencies": {},
+      "dependencies": {
```

Comment:
add these packages to here is for set `design-system` dependencies, otherwise, it will not auto-update dependencies to Bit and NPM.